### DISCUSSION TITLE
Rename Wiki to Docs

### DISCUSSION THREAD

**Discussion by ghost** [4ian/GDevelop#5227] [Enhancement ideas (and their technical discussions)]
I like the new wiki: https://wiki.gdevelop.io/ It looks and feels much better than the old one!

The new wiki seems to be using MkDocs to create the wiki pages from Markdown files in the Documentation repo: https://github.com/GDevelopApp/GDevelop-documentation This, to me, makes it less of a wiki and more of a documentation site. In other words, users cannot simply edit the wiki pages like they could previously. They now have to fork the repo, make their changes and then submit a PR.

I don't have any issues with this, but I would suggest renaming the site/URL from Wiki to Docs (e.g. https://docs.gdevelop.io/)...

I realise that URL currently redirects to the JavaScript game engine docs (https://docs.gdevelop.io/GDJS%20Runtime%20Documentation/) but I'm sure the wiki could be moved there too. It might even make the former more discoverable...

I looked around for the place where the discussion for the new wiki might have taken place (issue queue, PRs, documentation repo, discussions, forum), but couldn't find anything. So hopefully this is an appropriate place to suggest/discuss this.

**Comment by AlexandreSi** [comment]
We've been discussing this recently. Indeed wiki is not the best word to describe our docs. We have to think about how to make an easy transition to this sub domain.

**Reply by 4ian** [reply]
+1 with what AlexandreSi said, we should probably move to docs.gdevelop.io and do a redirection of the old domain.
